bin/exec.go

```diff
@@ -87,73 +87,45 @@ func main() {
+		// Get thee Default ENV's from ChaosExperiment
+		//perExperiment.Env = utils.GetEnvFromExperiment(engineDetails, engineDetails.Experiments[i])
+		perExperiment.SetDefaultEnv(clients)
 
 		log.Info("Printing the Default Variables", perExperiment.Env)
 
 		// Get the ConfigMaps for patching them in the job creation
 		log.Infoln("Find the configMaps in the chaosExperiments")
 
-		configMapExist, configMaps := utils.CheckConfigMaps(engineDetails, config, engineDetails.Experiments[i])
-
-		var validatedConfigMaps []v1alpha1.ConfigMap
-		var errorsListForConfigMaps []error
-		if configMapExist == true {
-			log.Infoln("Config Maps Found")
-			//fetch details and apply those config maps needed
-			// to be used in the job creation
-			// first convert the format of ConfigMap's Data to map[string]string
-			// & then use the kube-builder to build config maps
-			validatedConfigMaps, errorsListForConfigMaps = utils.ValidateConfigMaps(configMaps, engineDetails, clients)
-
-			if errorsListForConfigMaps != nil {
-				log.Errorf("Printing Errors, found while Validating ConfigMaps : %v, Will abort the Experiment Execution", errorsListForConfigMaps)
-				continue
-			}
-		} else {
-			log.Infoln("Unable to find ConfigMaps")
-		}
+		perExperiment.SetConfigMaps(clients)
 
-		secretsExist, secrets := utils.CheckSecrets(engineDetails, config, engineDetails.Experiments[i])
-
-		var validatedSecrets []v1alpha1.Secret
-		var errorsListForSecrets []error
-		if secretsExist == true {
-			log.Infoln("Secrets Found")
-			//fetch details and apply those config maps needed
-			// to be used in the job creation
-			// first convert the format of ConfigMap's Data to map[string]string
-			// & then use the kube-builder to build config maps
-			validatedSecrets, errorsListForSecrets = utils.ValidateSecrets(secrets, engineDetails, clients)
-
-			//log.Infoln("Printing VolumeMounts : ", volumeMounts)
-			if errorsListForSecrets != nil {
-				log.Errorf("Printing Errors, found while Validating Secrets : %v, Will abort the Experiment Execution", errorsListForSecrets)
-				continue
-			}
-		} else {
-			log.Infoln("Unable to find Secrets")
+		err := perExperiment.ValidateConfigMaps(clients)
+		if err != nil {
+			log.Infof("Aborting Execution")
+			return
 		}
 
-		log.Infof("Printing Validated ConfigMaps: %v", validatedConfigMaps)
-		log.Infof("Printing Validated Secrets: %v", validatedSecrets)
+		perExperiment.SetSecrets(clients)
+		err = perExperiment.ValidateSecrets(clients)
+		if err != nil {
+			log.Infof("Aborting Execution")
+			return
+		}
 
-		// 1. []*volume.Builder
-		volumeBuilders := utils.CreateVolumeBuilder(validatedConfigMaps, validatedSecrets)
-		//log.Infof("Printing volumeBuilders: %v", volumeBuilders)
+		log.Infof("Printing Validated ConfigMaps: %v", perExperiment.ConfigMaps)
+		log.Infof("Printing Validated Secrets: %v", perExperiment.Secrets)
```

Comment:
`Can you fix the log message`

Reply:
What to fix in this? It is perfectly fine.

Reply:
`Printing` word is not required